src/components/ReportWelcomeText.tsx

```diff
@@ -53,12 +55,22 @@ function ReportWelcomeText({report, policy, personalDetails}: ReportWelcomeTextP
         Navigation.navigate(ROUTES.REPORT_WITH_ID_DETAILS.getRoute(report.reportID));
     };
 
+    const getWelcomeHeroText = useMemo(() => {
```

Comment:
This is a memoized const, not a function:
```suggestion
    const welcomeHeroText = useMemo(() => {
```